### DISCUSSION TITLE
Change Origin point of DWG

### DISCUSSION THREAD

**Discussion by shamindra-adna** [mozman/ezdxf#1083] [Q&A]
Hi,
I am fetching coordinates of entities. While doing so, came through some entities whose coordinates are in negative space of the Coordinate System. How can I move the ORIGIN of DWG file so that all entities stays under positive XY plane?

**Comment by mozman** [comment]
You cannot move the origin of DXF files you have to move the entities.

**Reply by shamindra-adna** [reply]
Okay, thank you.
